radio/src/audio_arm.cpp

```diff
@@ -684,8 +695,12 @@ int ToneContext::mixBuffer(AudioBuffer * buffer, int volume, unsigned int fade)
     }
 
     for (int i=0; i<points; i++) {
+
       int16_t sample = sineValues[int(toneIdx)] / state.volume;
-      mixSample(&buffer->data[i], sample, fade);
+#if defined(SOFTWARE_VOLUME)
+      sample *= (float)currentSpeakerVolume / VOLUME_LEVEL_MAX ;
```

Comment:
This is bad, when you correct the sw volume math as suggested above, this will not be needed.